src/main/java/net/greatstart/dto/DtoUserProfile.java

```diff
@@ -14,4 +14,6 @@
     private String address;
     private String phoneNumber;
     private String email;
+    private String lastName;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "lastName" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)